### PR TITLE
Migrate Api Key Manangement tests

### PR DESCRIPTION
**Pull request by jrmccannon**
This moves the Api Key Management Api tests out of the endpoint test file to the new Integration tests project.  Also includes an addition to the Database seed method to add scopes to the public key that's created on startup.

### CODE REVIEW

File: src/Service/EventLog/Models/EventResponse.cs [hunk 1]
```diff
@@ -4,6 +4,8 @@ namespace Passwordless.Service.EventLog.Models;
 
 public class EventResponse
 {
+    public EventResponse() { }
+
     public EventResponse(DateTime performedAt, string message, string performedBy, string applicationId, EventType eventType, Severity severity, string subject, string apiKeyId)
```
Comment (Tyrrrz) [inline src/Service/EventLog/Models/EventResponse.cs:9]: If we already have `init` props, do we need the constructor at all?

File: tests/Api.IntegrationTests/Helpers/App/CreateAppHelpers.cs [hunk 2]
```diff
@@ -18,4 +20,7 @@ public static async Task<HttpResponseMessage> CreateApplication(this HttpClient
 
         return await client.PostAsJsonAsync($"/admin/apps/{applicationName}/create", AppCreateGenerator.Generate());
     }
+
+    public static Task<HttpResponseMessage> CreateApplication(this HttpClient client)
```
Comment (Tyrrrz) [inline tests/Api.IntegrationTests/Helpers/App/CreateAppHelpers.cs:24]: Should these method be suffixed by `Async`?

**Review by Tyrrrz — APPROVED** [review]
Looks great :)

File: src/Service/EventLog/Models/EventResponse.cs (hunk 1 above)
Comment (jrmccannon) [reply src/Service/EventLog/Models/EventResponse.cs:9]: The JsonSerializer was complaining about not being able to use the constructor.

**Pull request merged**